File: src/js/fizzbuzz.js
function fizzbuzz(limit) {
    for (let i = 1; i <= limit; i++) {
        let isDiv3;
        let isDiv5 = i%5==0;
        if((isDiv3 = (i%3==0)) && isDiv5)
            console.log("FizzBuzz");
        else if(isDiv3)
            console.log("Fizz");
        else if(isDiv5)
            console.log("Buzz");
        else if(!isDiv3)
            console.log(i);
        
    }
}

File: src/rust/fizzbuzz.rs
fn fizzbuzz(n: u64)
{
    for i in (0..=n)
    {
        if i%3==0 {
            print!("Fizz");
        }
        if i%5==0 {
            print!("Buzz");
        }
        else if i%3!=0 {
            print!("{}", i);
        }

        println!();
    }
}